modules/op-release-notes-1-10.adoc

```diff
@@ -197,4 +197,18 @@ Before this update, the Tekton Operator did not expose the performance configura
 * `buckets`
 * `kube-api-qps`
 * `kube-api-burst`
-* `threads-per-controller`
\ No newline at end of file
+* `threads-per-controller`
+
+[id="release-notes-1-10-4_{context}"]
+== Release notes for {pipelines-title} General Availability 1.10.4
+
+With this update, {pipelines-title} General Availability (GA) 1.10.4 is available on {product-title} 4.11, 4.12, and 4.13.
+
+[id="fixed-issues-1-10-4_{context}"]
+=== Fixed issues
+
+* This update fixes the bundle conversion issue for `v1beta1` pipeline references. Now, the bundle conversion feature converts the `PipelineRef` bundle to the `Resolver` bundle. 
```

Comment:
This update fixes the bundle resolver conversion issue for pipelineref field in PipelineRun. Now, the bundle conversion converts the `PipelineRef` bundle to the `Pipeline` kind in PipelineRun

Reply:
This update fixes the bundle resolver conversion issue for pipelineref field in PipelineRun. Now, it sets the kind field to `Pipeline` after conversion

Reply:
yeah it's was setting the kind field to `Task` before (even when resolving Pipeline).. which made thing act weird to not working in some cases.

Reply:
done